pagopa/io-app
# fix: [#175949490] Request current PIN before letting the user change it

## How to test
- Go to "Profile" tab
- Tap "Change the unlock code"
- You should be prompted to insert the current PIN
- Reload the app and access it with the new PIN
